### ISSUE TITLE
Issue with tutorials/nginx-reverse-proxy-docker.md

### ISSUE THREAD

**Issue by IanVaughan** [GoogleCloudPlatform/community#1804]
It jumps from creating an instance to running docker...
Its a little unclear, but as a guess, before `Setting up the reverse proxy`, we should connect via SSH to the new instance?

**Comment by ToddKopriva** [comment]
@tswast , could you have a look at this question about a document that you submitted?

**Comment by tswast** [comment]
Yes, these commands are assumed to be run from the instance which was created.

It's worth adding a step instructing the reader to SSH to the instance after it is created.

**Comment by ToddKopriva** [comment]
Thanks, @tswast .

**Comment by ToddKopriva** [comment]
Thanks, @IanVaughan , for pointing this out.
